conf/application.conf

```diff
@@ -147,7 +147,15 @@ external-url {
 }
 
 services-config {
-  list = "atar, gagmr, route1, ctc"
+  list = "cds,atar,gagmr,route1,ctc"
```

Comment:
We shouldn't add CDS to local, at least not now. This config can leak to the prod config if it is not overridden

Reply:
So what should I do, just leave it out the list?

Reply:
took cds out of the services-config.list as is the case in production and left the rest of the cds config in this file and tested the app. Got page not found as expected